docs/en_US/Tutorial/InstallCustomizedAlgos.md

```diff
@@ -0,0 +1,164 @@
+**How to install customized algorithms as builtin tuners, assessors and advisors**
+===
+
+## Overview
+
+NNI provides a lot of [builtin tuners](../Tuner/BuiltinTuner.md), [advisors](../Tuner/BuiltinTuner.md#Hyperband) and [assessors](../Assessor/BuiltinAssessor.md) can be used directly for Hyper Parameter Optimization, and some extra algorithms can be installed via `nnictl package install --name <name>` after NNI is installed. You can check these extra algorithms via `nnictl package list` command.
+
+NNI also provides the ability to build your own [customized tuners](../Tuner/CustomizeTuner.md), [advisors](../Tuner/CustomizeAdvisor.md) and [assessors](../Assessor/CustomizeAssessor.md).
```

Comment:
better to append at the end of this paragraph: 
To use the customized algorithm, users can simply follow the spec in experiment config file to properly reference the algorithm, which has been illustrated in the [tutorials of customized tuners/advisors/assessors]().

Reply:
Thanks,  fixed.